tests/Utils/SuggestionListTest.php

```diff
@@ -7,38 +7,90 @@
 
 class SuggestionListTest extends TestCase
 {
-    // DESCRIBE: suggestionList
-
     /**
+     * @see describe('suggestionList')
      * @see it('Returns results when input is empty')
      */
     public function testResturnsResultsWhenInputIsEmpty(): void
     {
-        self::assertEquals(
-            Utils::suggestionList('', ['a']),
-            ['a']
-        );
+        self::assertEquals(Utils::suggestionList('', ['a']), ['a']);
```

Comment:
Expected goes first, and I would prefer `assertSame()` here since order matters

Reply:
This particular test was like that before, I only changed formatting, but you are right nonetheless. I have fixed it everywhere.

The different order of parameters in _assertions_ and _expectations_ is pretty annoying. Have you considered using https://github.com/pestphp/pest for this project? It would allow us to keep the same structure of tests as in the reference implementation.

Reply:
Great idea, happy to try - see https://github.com/webonyx/graphql-php/issues/1086